### PR TITLE
fix(theme-translations): remove redundant navigation text in aria label

### LINKED ISSUE #8834: Redundant 'navigation' text in aria-label for Docs pages navigation
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#issues).

### Prerequisites

- [X] I'm using the latest version of Docusaurus.
- [X] I have tried the `npm run clear` or `yarn clear` command.
- [X] I have tried `rm -rf node_modules yarn.lock package-lock.json` and re-installing packages.
- [X] I have tried creating a repro with https://new.docusaurus.io.
- [X] I have read the console error message carefully (if applicable).

### Description

On the pagination nav, the aria-label attribute is set as "Docs pages navigation." The word navigation is redundant, since it appears inside a \<nav\> element. This results in it being read out loud like "Docs pages **navigation navigation** region" or something similar. 

Just for context: other navigation elements on the page seem to take this into account already, for example the "Pattern navigation" aria-label is just "Pattern"

### Reproducible demo

_No response_

### Steps to reproduce

1. Start JAWS
2. Navigate to https://docusaurus.io/docs/next
3. Tab to the "Docs pages navigation navigation region"

Alternatively:

1. Navigate to https://docusaurus.io/docs/next
2. Scroll to bottom of page
3. Inspect "Next Getting Started" page element
4. See aria-label on `<nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation">`

### Expected behavior

The region should announce as "Docs pages navigation region"

### Actual behavior

"Docs pages navigation **navigation** region" (e.g. navigation is repeated)

### Your environment

- Public source code: https://github.com/facebook/docusaurus
- Public site URL: https://docusaurus.io/
- Docusaurus version used: 2.4.0
- Environment name and version: Chrome 111 and JAWS

### Self-service

- [X] I'd be willing to fix this bug myself.

### PR DESCRIPTION
**Pull request by tarunrajput**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

Closes #8834

## Test Plan

Tested using a screen reader
image

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

## Related issues/PRs

#8834

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Could you try to update some of the localization messages, and revert other messages to the English one?

File: packages/docusaurus-theme-translations/locales/base/theme-common.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
   "theme.docs.breadcrumbs.home___DESCRIPTION": "The ARIA label for the home page in the breadcrumbs",
   "theme.docs.breadcrumbs.navAriaLabel": "Breadcrumbs",
   "theme.docs.breadcrumbs.navAriaLabel___DESCRIPTION": "The ARIA label for the breadcrumbs",
-  "theme.docs.paginator.navAriaLabel": "Docs pages navigation",
+  "theme.docs.paginator.navAriaLabel": "Docs pages",
```
Comment (Josh-Cena) [inline packages/docusaurus-theme-translations/locales/base/theme-common.json:86]: Could you try to update some of the localization messages, and for the rest that you are not sure, revert them to the English one?

Comment (tarunrajput) [reply packages/docusaurus-theme-translations/locales/base/theme-common.json:86]: Sure, I've updated the translations

**Comment by slorber** [comment]
Thanks 👍 

Will be hard to review properly but assuming it's good enough. If you notice any error please submit a new PR

**Pull request merged**